types/message.go

```diff
@@ -21,22 +24,29 @@ type GasUnits = Uint64
 // BlockGasLimit is the maximum amount of gas that can be used to execute messages in a single block
 var BlockGasLimit = NewGasUnits(10000000)
 
+// EmptyMessagesCID is the cid of an empty collection of messages.
+var EmptyMessagesCID cid.Cid
+
+// EmptyReceiptsCID is the cid of an empty collection of receipts.
+var EmptyReceiptsCID cid.Cid
```

Comment:
There are a few places you might be able to use this and don't during genesis construction assuming those functions can take types as dependency.